schema/me.js

```diff
@@ -22,7 +27,24 @@ const Me = new GraphQLObjectType({
       resolve: ({ default_profile_id }) =>
         gravity(`profile/${default_profile_id}`),
     },
-    bidder_positions: BidderPositions,
+    bidder_positions: {
+      type: new GraphQLList(BidderPosition.type),
+      description: "A list of the current user's bidder positions",
+      args: {
+        current: {
```

Comment:
`/api/v1/me/bidder_positions` returns all bids (including historical bids). For instance if Devang bids $10k on a work, then later ups his max bid to $20k—it will create 2 bidder positions, but the $10k one is the actually relavant one, until someone bids higher and the 20k bid outbids them. The way it's implemented now it can create a weird "historical" view of bids (see below). This `current` arg will allow one to filter the bidder positions to _only_ the most recent/relevant bids. This is what tripped me up yesterday and cause me to pend the my active bids.

![image](https://cloud.githubusercontent.com/assets/555859/12958329/238ab478-cfff-11e5-8d6b-86520c31a515.png)